Sources/PackageDescription/PackageRequirement.swift

```diff
@@ -192,10 +205,12 @@ extension Package.Dependency {
     /// versions, and may require you to modify your code when they update.
     /// The version rule requires Swift packages to conform to semantic
     /// versioning. To learn more about the semantic versioning standard,
-    /// visit [semver.org](https://semver.org).
+    /// visit the [Semantic Versioning 2.0.0](https://semver.org) website.
     @available(_PackageDescription, introduced: 999)
     public enum RegistryRequirement {
+        /// A version based requirement.
```

Comment:
Since the other case is also a version-based requirement, perhaps it's more accurate to highlight the exact-ness of this case.